hack/setup-monitoring.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+
+# ----------------------------------------------------------------
+# OAuth2 Proxy Secrets
+# ----------------------------------------------------------------
+
+oauth2-secret() {
+  if [[ $# -ne 4 ]]; then
+    echo "invalid number of arguments"
+    echo "usage:"
+    echo "  $0 oauth2-secret NAME CLIENT_ID CLIENT_SECRET COOKIE_SECRET"
+    exit 1
+  fi
+
+  NAME=$1
+  CLIENT_ID=$2
+  CLIENT_SECRET=$3
+  COOKIE_SECRET=$4
+
+  oc create secret generic $NAME \
```

Comment:
would it make sense to use `apply` instead of `create` - in case it needs to be replaced with new values?

Reply:
see c3454ee